lib/fdotypes.c

```diff
@@ -1364,8 +1337,7 @@ fdo_public_key_t *fdo_public_key_read(fdor_t *fdor)
 	case FDO_CRYPTO_PUB_KEY_ENCODING_X5CHAIN:
 		LOG(LOG_ERROR, "Invalid PublicKey: pkEnc.X5CHAIN is not supported.\n");
 		goto err;
-	case FDO_CRYPTO_PUB_KEY_ENCODING_COSEKEY:
-		;
+	case FDO_CRYPTO_PUB_KEY_ENCODING_COSEKEY: ;
```

Comment:
Why you need ";"

Reply:
Labels (FDO_CRYPTO_PUB_KEY_ENCODING_COSEKEY) can only be followed by statements, and declarations do not count as statements in C. So ';' is used to make an empty statement here.